src/Model/Componentfactory/include/Componentfactory.hpp

```diff
@@ -178,7 +178,8 @@ namespace Model::Componentfactory {
         auto opt_boundary_schema = ConcreteNode::get_boundary_schema();
         if (opt_boundary_schema.has_value()) {
           Aux::schema::add_defaults(
-              opt_boundary_schema.value(), this->defaults);
+              opt_boundary_schema.value(),
+              this->defaults.value("boundary_values", R"({})"_json));
```

Comment:
Since unused defaults result in a runtime error, we can not just use the same defaults from the topology json for the boundary, control, initial jsons since these schemas have different keys. Therefore I am moving the defaults for those into a subcategory.

Reply:
cf. https://github.com/eike-fokken/grazer/runs/3244210820